Tools/src/FormatCheckTool/FormatCheckTool.cs

```diff
@@ -0,0 +1,234 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.Recognizers.Text;
+using Microsoft.Recognizers.Text.Choice;
+using Microsoft.Recognizers.Text.DateTime;
+using Microsoft.Recognizers.Text.Number;
+using Microsoft.Recognizers.Text.NumberWithUnit;
+using Microsoft.Recognizers.Text.Sequence;
+using Newtonsoft.Json;
+using Microsoft.Recognizers.Text.DataDrivenTests;
+
+namespace FormatCheckTool
+{
+    class FormatCheckTool
+    {
+        private static readonly String PleaseCheckStr = "Please Check it.";
+
+        static void Main(string[] args)
+        {
+            // Encoding for 'exotic' characters e.g. '€'
+            Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
+
+            ShowIntro();
+
+            while (true)
+            {
+                Console.WriteLine("Enter the specs JSON file path to be checked:");
+                var input = Console.ReadLine()?.Trim();
+                Console.WriteLine();
+
+                if (input?.ToLower(CultureInfo.InvariantCulture) == "exit")
```

Comment:
No need for this as the tool for now won't be interactive. I.e. one run per spec file or set of spec files.